tests/testthat/test_all_open.R

```diff
@@ -3,21 +3,80 @@ context("all open intervals\n")
 library(iregnet)
 
 set.seed(1)
-toy.features <- matrix(rnorm(10), 5, 2)
+toy.features <- matrix(rnorm(10, 1, 0.1), 5, 2)
 toy.targets <- rbind(
   c(1, 2),
   c(-Inf, 5),
-  c(1, Inf),
+  c(3, Inf),
   c(-Inf, 3),
   c(-Inf, 4))
+if(interactive()){
+  colnames(toy.targets) <- c("min.L", "max.L")
+  toy.df <- data.frame(toy.targets, toy.features)
+  library(survival)
+  fit <- survreg(
```

Comment:
this survreg code will only work if you install the new survival from github. You would have to clone the git repos, then `cd survival/noweb; make fun; cd ../..;R CMD build survival`